week0/homework/Person.js

```diff
@@ -0,0 +1,22 @@
+
+module.exports = class Person {
+  constructor (name,age) {
+    this.name = name;
+    this.age = age;
+  }
+  addPhone(phoneNum) {
+    this.phoneNum = phoneNum;
+  }
+
+  call() {
+    if (!this.phoneNum) {
+      return  "Calling" + this.name + "at" + this.phoneNum;
+    }else{
+       return this.name + "has no phone number saved.";
+}
+  }
+
+birthday() {
```

Comment:
Remember, template strings in ES5/6 are much better to use for string construction: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals